src/api/index.ts

```diff
@@ -1,27 +1,28 @@
 import {AptosClient, Types} from "aptos";
+import {isHex} from "../pages/utils";
 import {withResponseError} from "./client";
 
 export async function getTransactions(
-  requestParameters: {start?: number; limit?: number},
+  requestParameters: {start?: bigint; limit?: number},
```

Comment:
JS numbers can safely represent integers up to 2^53 - 1 (roughly 9 quadrillion). I assume the SDK type changed, but we can keep our code the same without having to worry about BigInt by keeping the `getTransactions` signature the same and performing the number -> BigInt conversion inside it (maybe with an assertion that `0 <= start <= Number.MAX_SAFE_INTEGER`).